<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="11" hostname="DESKTOP-K0SFL7M" failures="3" tests="7" name="Default test" time="353.165" errors="0" timestamp="2021-10-08T19:21:17 PKT">
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="NavigateOM" time="11.586"/>
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="OrganizationSiteMandatoryCheck" time="19.715"/>
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="SiteResetButton" time="2.192"/>
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="CreateOrganizationSite" time="36.569">
    <failure type="java.lang.Exception" message="java.lang.AssertionError: The following asserts failed:
	expected [USA] but found [],
	expected [North Carolina] but found [],
	expected [Lincolnton] but found []">
      <![CDATA[java.lang.Exception: java.lang.AssertionError: The following asserts failed:
expected [USA] but found [],
expected [North Carolina] but found [],
expected [Lincolnton] but found []
at Test.Ancera.Administration.OrganizationManagement.CreateOrganizationSite(OrganizationManagement.java:999)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: The following asserts failed:
expected [USA] but found [],
expected [North Carolina] but found [],
expected [Lincolnton] but found []
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Test.Ancera.Administration.OrganizationManagement.CreateOrganizationSite(OrganizationManagement.java:991)
... 28 more
]]>
    </failure>
  </testcase> <!-- CreateOrganizationSite -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="UpdateOrganizationSites" time="7.509">
    <failure type="java.lang.Exception" message="java.lang.AssertionError: expected [USA] but found []">
      <![CDATA[java.lang.Exception: java.lang.AssertionError: expected [USA] but found []
at Test.Ancera.Administration.OrganizationManagement.UpdateOrganizationSites(OrganizationManagement.java:1053)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: expected [USA] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Test.Ancera.Administration.OrganizationManagement.UpdateOrganizationSites(OrganizationManagement.java:1034)
... 28 more
]]>
    </failure>
  </testcase> <!-- UpdateOrganizationSites -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="VerifyUpdateOrganizationSites" time="180.524">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for visibility of element located by By.xpath: /html/body/app-root/div/app-manage-organization/app-popup-component/div/div/div/div[3]/app-create-site-component/form/div[1]/div/div[1]/div/ul/div/li/ul/li/div/div[2]/span (tried for 180 second(s) with 500 milliseconds interval)
Build info: version: &amp;apos;4.0.0-alpha-7&amp;apos;, revision: &amp;apos;de8579b6d5&amp;apos;
System info: host: &amp;apos;DESKTOP-K0SFL7M&amp;apos;, ip: &amp;apos;192.168.56.1&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;15.0.2&amp;apos;
Driver info: org.openqa.selenium.support.events.EventFiringWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 94.0.4606.71, chrome: {chromedriverVersion: 93.0.4577.63 (ff5c0da2ec0ad..., userDataDir: C:\Users\User\AppData\Local...}, goog:chromeOptions: {debuggerAddress: localhost:53400}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 9893d68e0f61ad44e6b8a566d45016de">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: /html/body/app-root/div/app-manage-organization/app-popup-component/div/div/div/div[3]/app-create-site-component/form/div[1]/div/div[1]/div/ul/div/li/ul/li/div/div[2]/span (tried for 180 second(s) with 500 milliseconds interval)
Build info: version: '4.0.0-alpha-7', revision: 'de8579b6d5'
System info: host: 'DESKTOP-K0SFL7M', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '15.0.2'
Driver info: org.openqa.selenium.support.events.EventFiringWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 94.0.4606.71, chrome: {chromedriverVersion: 93.0.4577.63 (ff5c0da2ec0ad..., userDataDir: C:\Users\User\AppData\Local...}, goog:chromeOptions: {debuggerAddress: localhost:53400}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 9893d68e0f61ad44e6b8a566d45016de
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:211)
at Test.Ancera.Administration.OrganizationManagement.VerifyUpdateOrganizationSites(OrganizationManagement.java:1086)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyUpdateOrganizationSites -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="DeleteOrganizationSites" time="47.939"/>
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="InActiveOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- InActiveOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="SearchOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- SearchOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="ClosePopup" time="0.0">
    <ignored/>
  </testcase> <!-- ClosePopup -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="ResetButton" time="0.0">
    <ignored/>
  </testcase> <!-- ResetButton -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="DeleteOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- DeleteOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="SearchUpdateOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- SearchUpdateOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="MandatoryFieldCheck" time="0.0">
    <ignored/>
  </testcase> <!-- MandatoryFieldCheck -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="VerifyInActiveOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyInActiveOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="UpdateOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- UpdateOrganization -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="OrganizationSitesCheck" time="0.0">
    <ignored/>
  </testcase> <!-- OrganizationSitesCheck -->
  <testcase classname="Test.Ancera.Administration.OrganizationManagement" name="CreateOrganization" time="0.0">
    <ignored/>
  </testcase> <!-- CreateOrganization -->
</testsuite> <!-- Default test -->
